static/js/settings_users.js

```diff
@@ -497,7 +496,7 @@ function handle_bot_deactivation($tbody) {
         const $button_elem = $(e.target);
         const $row = $button_elem.closest(".user_row");
         const bot_id = Number.parseInt($row.attr("data-user-id"), 10);
-        
+
```

Comment:
The first commit adds trailing whitespace, which the second commit removes, presumably because it failed the linter. It's  important to rebase and fix issues like this in the commit that introduced them, since otherwise CI will fail if we merge only the first commit. (and `git bisect` may be messed up, etc.). Every commit should individually pass lint, and it takes some care to ensure that.